src/core/qgsvectorlayer.h

```diff
@@ -2115,6 +2124,11 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
      */
     void setAllowCommit( bool allowCommit ) SIP_SKIP;
 
+    /**
+     * Returns true if the features were counted.
+     * \since QIGS 3.8
+     */
+    bool featuresCounted() const { return mSymbolFeatureCounted; }
```

Comment:
That's only used in tests right now, correct?
Kill it!

Reply:
done,  I just thought exposing might be nice for some other uses but alright.